infrastructure/gsuite-exporter/gsuite_exporter/cli.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import importlib
+import re
+import sys
+import logging
+from gsuite_exporter import exporters
+from gsuite_exporter.collectors.reports import AdminReportsAPIFetcher
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logging.getLogger('googleapiclient').setLevel(logging.ERROR)
+
+logger = logging.getLogger(__name__)
+
+def get_exporter_cls(exporter_class):
+    try:
+        file, name = tuple(exporter_class.split('.'))
+        package = 'gsuite_exporter.exporters.%s' % file
+        return getattr(importlib.import_module(package), name)
+    except Exception as e:
+        logger.exception(
+            'Exporter "%s" not found. Verify the exporter class name is'
+            'prefixed by the exporter file name.\n'
+            'Example: `stackdriver_exporter.StackdriverExporter`',
+            exporter_class)
+        sys.exit()
```

Comment:
Sure you want to exit `0` after this exception?

Reply:
Good catch, changed to sys.exit(1) to mark an error